src/app/modules/consultations/consultation-profile/consultation-response-text/consultation-response-text.component.ts

```diff
@@ -147,7 +156,6 @@ export class ConsultationResponseTextComponent implements OnInit, AfterViewCheck
 
   enableSubmitResponse(value) {
```

Comment:
@Nishanthh Bro is this being used anymore?

Reply:
@avikt yes bro, function name has been renamed conventionally bro